serverless.yml

```diff
@@ -199,6 +199,50 @@ functions:
               - name: request.path.hash
               - name: request.header.origin
 
+  get_token_balances_handler:
+    handler: handlers/token_balance_api.get_token_balances
+    timeout: 6
+    maximumRetryAttempts: 0
+    package:
+      patterns:
+        - 'handlers/token_balance_api.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: token_balances
+          method: get
+          cors: true
+          caching:
+            enabled: true
+            ttlInSeconds: 300
+            cacheKeyParameters:
+              - name: request.queryString.token_id
+              - name: request.querystring.sort_by
+              - name: request.querystring.order
+              - name: request.querystring.search_after
+              - name: request.header.origin
+
+  get_token_balance_information_handler:
+    handler: handlers/token_balance_api.get_token_information
+    timeout: 6
+    maximumRetryAttempts: 0
+    package:
+      patterns:
+        - 'handlers/token_balance_api.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: token_balances/information
```

Comment:
maybe token_balances/{id} would be sufficient here? I do not know if the word `information` aggregate here as API name.

Reply:
I think `token_balances/{id}` feels like it is a token balance for a given token_id

The idea is to have some information about token_balances for a given token_id, that's why I used `token_balances`/`information`